assets/src/domain/eventEditor/services/context/EntityListContext/withEntityListContext.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const withEntityListContext = (
+	Provider: React.ReactType,
+	Component: React.ReactType,
+	props?: any
```

Comment:
can this be improved a bit? it's confusing a bit

Reply:
it's clear what it does, but the style is a bit confusing

Reply:
Any suggestions?

Reply:
added above